code/modules/antagonists/blob/blob/blobs/blob_mobs.dm

```diff
@@ -142,8 +142,10 @@
 	// Create the reagents to put into the air
 	create_reagents(10)
 
-	if(overmind && overmind.blob_reagent_datum)
-		reagents.add_reagent(overmind.blob_reagent_datum.id, 10)
+
+
+	if(overmind && overmind.blobstrain)
+		overmind.blobstrain.on_sporedeath(src)
```

Comment:
whole thing should probably be overridable but i guess that can be done later.

Reply:
yeah this is more about moving the reagent stuff into a better datum, i'll do the other cleanup later / someone else can do it